src/elements/form/FieldInteractionApi.ts

```diff
@@ -437,7 +437,8 @@ export class FieldInteractionApi {
     public setLoading(key: string, loading: boolean) {
         let control = this.getControl(key);
         if (control) {
-            if (loading) {
+            if (loading && control.value) {
```

Comment:
Need to check the value here? If the user nulls our the field we still probably want to perform the async too

Reply:
I added this because the animation was going when you wipe out the field. It looked weird to me. I can take out the check if that's the desired behavior (including the loading animation) or I can add another check in the ngIf in template to check for value there and only show the loading animation if the field has a value.

Reply:
Actually, now that I think about it, I probably don't need this. An async picker field would still need to load when you wipe out the value to get all the options, etc.